Gemfile.lock

```diff
@@ -496,6 +500,13 @@ GEM
     simplecov_json_formatter (0.1.2)
     skylight (5.0.1)
       activesupport (>= 5.2.0)
+    slim (4.1.0)
+      temple (>= 0.7.6, < 0.9)
+      tilt (>= 2.0.6, < 2.1)
+    slim-rails (3.2.0)
+      actionpack (>= 3.1)
+      railties (>= 3.1)
+      slim (>= 3.0, < 5.0)
```

Comment:
Bit annoying that `govuk-components` adds `slim` as a dependency – is this needed? @duncanjbrown

Reply:
Maybe not for us, but not our choice either

Reply:
Will update this PR with the correct version of this gem when https://github.com/DFE-Digital/govuk-components/pull/123 is merged.